task/03-date-tasks.js

```diff
@@ -56,7 +56,10 @@ function parseDataFromIso8601(value) {
  *    Date(2015,1,1)    => false
  */
 function isLeapYear(date) {
-   throw new Error('Not implemented');
+   var buff = new Date(date);
+   var year = buff.getFullYear();  
+   return (year % 100 === 0) ? (year % 400 === 0) : (year % 4 === 0)
```

Comment:
`var` is deprecated. It has no sense to use even for browsers code currently.

Reply:
done